netty/src/main/java/io/atomix/catalyst/transport/NettyProperties.java

```diff
@@ -36,11 +37,11 @@
 
   public static final String SSL_ENABLED = "ssl.enabled";
   public static final String SSL_PROTOCOL = "ssl.protocol";
-  public static final String SSL_TRUSTSTORE_PATH = "ssl.truststore.path";
-  public static final String SSL_TRUSTSTORE_PASSWORD = "ssl.truststore.password";
-  public static final String SSL_KEYSTORE_PATH = "ssl.keystore.path";
-  public static final String SSL_KEYSTORE_PASSWORD = "ssl.keystore.password";
-  public static final String SSL_KEYSTORE_KEY_PASSWORD = "ssl.keystore.keypassword";
+  public static final String SSL_TRUST_STORE_PATH = "ssl.trustStore.path";
+  public static final String SSL_TRUST_STORE_PASSWORD = "ssl.trustStore.password";
+  public static final String SSL_KEY_STORE_PATH = "ssl.keyStore.path";
+  public static final String SSL_KEY_STORE_PASSWORD = "ssl.keyStore.password";
+  public static final String SSL_KEY_STORE_KEY_PASSWORD = "ssl.keyStore.keyPassword";
```

Comment:
For consistency with other existing properties in Atomix and Catalyst, I thought it made sense to make `trustStore` and `keyStore` camel case